src/fileimpl.h

```diff
@@ -145,6 +145,8 @@ namespace zim
       const std::string& getMimeType(uint16_t idx) const;
 
       std::string getChecksum();
+
+      #define CHUNK_SIZE 1024
```

Comment:
`CHUNK_SIZE` is not used elsewhere than `fileimpl.cpp`. No need to define it in the header.